android/gltfio-android/src/main/cpp/ResourceLoader.cpp

```diff
@@ -52,10 +52,21 @@ Java_com_google_android_filament_gltfio_ResourceLoader_nAddResourceData(JNIEnv*
         jlong nativeLoader, jstring url, jobject javaBuffer, jint remaining) {
     ResourceLoader* loader = (ResourceLoader*) nativeLoader;
     AutoBuffer* buffer = new AutoBuffer(env, javaBuffer, remaining);
-    const char* curl = env->GetStringUTFChars(url, nullptr);
-    loader->addResourceData(curl,
+    const char* cstring = env->GetStringUTFChars(url, nullptr);
+    loader->addResourceData(cstring,
             ResourceLoader::BufferDescriptor(buffer->getData(), buffer->getSize(), &destroy,
                     buffer));
+    env->ReleaseStringUTFChars(url, cstring);
```

Comment:
You make a copy of the string addResourceData right? Otherwise this will just free the string :)

Reply:
Yes :)